lib/fog-local.rb

```diff
@@ -1 +0,0 @@
-require 'fog/local'
```

Comment:
Ah, that's nice. I didn't know that bundler tries to replace `'-'` with `'/'` if require by plain gem name fails. So yes, this file is redundant and assuming rescuing from `LoadError` is cheap enough can be removed indeed (if you're letting Bundler autorequire gems you probably won't notice the difference...).